src/frontend/komponenter/Fagsak/Vilkårsvurdering/GeneriskVilkår/generisk-vilkår.less

```diff
@@ -1,6 +1,9 @@
 .generisk-vilkår {
-    width: 30rem;
-    padding: 1rem 0;
+    max-width: 60rem;
+    margin-top: 2rem;
+    & ~ .generisk-vilkår {
```

Comment:
Finnes ingen "first-of-class", som -child og -class). Må derfor velge de som har sibling av klassen før for å skille på første og resten.